src/component/CmpCollapsible.tsx

```diff
@@ -40,7 +41,7 @@ export class CmpCollapsible extends Component<Props, State> {
         super(props);
 
         this.state = {
-            collapsed: false,
+            collapsed: !!props.expandedByDefault, // TODO: collapsed has the wrong meaning. Should be the opposite.
```

Comment:
I'll handle this issue in a future PR.

Reply:
I want to keep the collapsed keyboard because the component is called CmpCollapsible and it makes it easier to understand for anyone looking at the code in the future.